src/librustdoc/html/static/main.js

```diff
@@ -342,6 +342,18 @@
                 }
             }
 
+            function findArg(obj, val) {
+                if (!obj || !obj.type || obj.type.inputs.length === 0) {
+                    return false;
+                }
+                for (var i = 0; i < obj.type.inputs.length; i++) {
+                    if (obj.type.inputs[i].name === val) {
+                        return true;
+                    }
+                }
+                return false;
```

Comment:
You replace the whole for thing with

```js
return obj.type.inputs.some(function (x) { return x.name === name; })
```

but I'm not sure if we still need to support browsers which don't have `Array.prototype.some`.

Reply:
That's the point. I'm supporting as much browsers as I can.